### DISCUSSION TITLE
Sync calendar with phone?

### DISCUSSION THREAD

**Discussion by lpossamai** [modoboa/modoboa#2220] [Help] [answered]
Is it possible to sync the calendar with my phone?

Cheers!

**Comment by tonioo** [comment, marked as answer]
Yes it is, you can use a caldav client like DavX5.

**Reply by lpossamai** [reply]
Thanks. Documentation is missing steps for that. [This](https://github.com/modoboa/modoboa-radicale/issues/72#issuecomment-814657964) helped.
